### PR TITLE
refactor: updated bottomsheet of multimeter

### LINKED ISSUE #1306: Update bottomsheet of multimeter
**Actual Behaviour**

The bottom sheet of multimeter doesn't contains information of the sections in the UI

**Expected Behaviour**

It should contain all information for user-friendliness.

**Steps to reproduce it**
'
Go to multimeter activity

**LogCat for the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1306 

**Changes**:updated bottomsheet of multimeter

**Screenshot/s for the changes**: [Add screenshot/s of the layout where you made changes or a `*.gif` containing a demonstration]
 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]
[bottomsheet.apk.zip](https://github.com/fossasia/pslab-android/files/2259807/bottomsheet.apk.zip)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -826,7 +821,14 @@
 
     <string name="multimeter_dialog_heading">Multimeter</string>
     <string name="multimeter_dialog_text">Use PV1, PV2, PV3 Connect the SEN and CAP pins to the resistor and capacitor respectively as shown in the figure to measure the resistance or capacitance of the connected element.</string>
-    <string name="multimeter_dialog_description">\u2022 To measure the voltage directly connect the power source(eg PV1,PV2 and PV3) to the channel pins as shown in the figure.\n\n\u2022 Similarly to measure frequency or count pulse connect SQ1,SQ2,SQ3 or SQ4 pins to ID1,ID2,ID3 or ID4.</string>
+    <string name="multimeter_dialog_description">\u2022 To measure the voltage directly connect the power source(eg PV1,PV2 and PV3) to the channel pins as shown in the figure.\n\n\u2022 Similarly to measure frequency or count pulse connect SQ1,SQ2,SQ3 or SQ4 pins to ID1,ID2,ID3 or ID4.\n\n
+        <b>Different Sections</b>\n\n
+        <b>Resistance:</b> Measures the resistance of the element connected to the SEN pin of the PSLab device.\n\n
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Shouldn't it be "element connected between SEN pin and GND pins"?

Comment (abhinavraj23) [reply app/src/main/res/values/strings.xml]: Yes @CloudyPadmal I will make the change :+1:

**Comment by abhinavraj23** [comment]
@CloudyPadmal Changes done as requested, @harsh-2711 @Avjeet please review

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -826,7 +821,14 @@
 
     <string name="multimeter_dialog_heading">Multimeter</string>
     <string name="multimeter_dialog_text">Use PV1, PV2, PV3 Connect the SEN and CAP pins to the resistor and capacitor respectively as shown in the figure to measure the resistance or capacitance of the connected element.</string>
-    <string name="multimeter_dialog_description">\u2022 To measure the voltage directly connect the power source(eg PV1,PV2 and PV3) to the channel pins as shown in the figure.\n\n\u2022 Similarly to measure frequency or count pulse connect SQ1,SQ2,SQ3 or SQ4 pins to ID1,ID2,ID3 or ID4.</string>
+    <string name="multimeter_dialog_description">\u2022 To measure the voltage directly connect the power source(eg PV1,PV2 and PV3) to the channel pins as shown in the figure.\n\n\u2022 Similarly to measure frequency or count pulse connect SQ1,SQ2,SQ3 or SQ4 pins to ID1,ID2,ID3 or ID4.\n\n
+        <b>Different Sections</b>\n\n
+        <b>Resistance:</b> Measures the resistance of the element connected between the SEN and GND pin of the PSLab device.\n\n
+        <b>Capacitance:</b> Measures the capacitance of the element connected to the CAP pin of the PSLab device.\n\n
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: This should also be changed like that, between CAP and GND

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -826,7 +821,14 @@
...
+        <b>Voltage:</b> Measures the voltage of any elements connect across any of the channel pins and other pins such as AN8.\n\n
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Similarly in this and the rest. We have to connect one end to measuring pin and the other end to GND

**Comment by abhinavraj23** [comment]
@CloudyPadmal Changes done :+1: @harsh-2711 @Avjeet Please review :+1:

**Pull request merged**